triggers/service/BUILD.bazel

```diff
@@ -129,7 +129,7 @@ common_binary_deps = [
 
 binary_deps = {
     "ce": common_binary_deps,
-    "ee": common_binary_deps + ["//release:ee-license"],
+    "ee": common_binary_deps,
 }
```

Comment:
Should we clean this up and just use `common_binary_deps` everywhere, if there aren't any differences anymore?